tools/topology/topology2/include/pipelines/cavs/passthrough-capture-be.conf

```diff
@@ -0,0 +1,68 @@
+#
+# BE DAI passthrough pipeline for capture
+#
+# All attributes defined herein are namespaced by alsatplg to
+# "Object.Pipeline.passthrough-be-capture.N.attribute_name"
+#
+# Usage: mixin-playback pipeline object can be instantiated as:
+#
+# Object.Pipeline.dai-pipeline."N" {
+#	direction	"capture"
+# 	period		1000
+# 	time_domain	"timer"
+# 	channels	2
+# 	rate		48000
+# }
+#
+# Where N is the unique pipeline ID within the same alsaconf node.
+#
+
+<include/common/audio_format.conf>
+<include/components/copier.conf>
+<include/components/pipeline.conf>
+
+Class.Pipeline."passthrough-capture-be" {
```

Comment:
@kv2019i can we call this 32-bit-capture.conf explicitly?

Reply:
yeah, lets give users a choice if possible. If not, we can hard code this later.